### PR TITLE
UI/e2e test

### PR DESCRIPTION
**Pull request by bartsaintgermain**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Build operator in release mode
         run: dotnet build -c Release
         working-directory: src/
-
+
```
Comment (qmfrederik) [inline .github/workflows/build.yaml:46]: nit: whitespace

File: .github/workflows/ui-test.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Provision CI environment
+        run: |
+          sudo apt-get update
+          ansible-galaxy install geerlingguy.nodejs
```
Comment (qmfrederik) [inline .github/workflows/ui-test.yaml:19]: I guess this is not really needed?

**Review by qmfrederik — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (bartsaintgermain) [reply .github/workflows/build.yaml:46]: seems there was a missing whitespace

File: .github/workflows/ui-test.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+jobs:
+  frontend:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Provision CI environment
```
Comment (qmfrederik) [inline .github/workflows/ui-test.yaml:16]: Do we need this?

**Review by qmfrederik — APPROVED** [review]

**Pull request merged**